# message for feature sharing

float64   x
float64   y
uint8[] descriptor
